docs/Changelog.md

```diff
@@ -32,7 +32,12 @@
 * [#7341](https://github.com/TouK/nussknacker/pull/7341) Added possibility to choose presets and define lists for Integer typed parameter inputs in fragments.
 * [#7356](https://github.com/TouK/nussknacker/pull/7356) Integers converted to BigDecimals have scale 18,
   this fixes issue with unexpected low scale when performing division on BigDecimals which were created in such conversion.
-* [#7368](https://github.com/TouK/nussknacker/pull/7368) Component rename: `periodic` to `sample-generator`
+* Changes to `periodic` / `sample-generator` component:
```

Comment:
Instead of slash I would add `(new `sample-gerator)` or renamed to

Reply:
done :+1: